be/CMakeLists.txt

```diff
@@ -491,6 +491,100 @@ if ("${USE_STAROS}" STREQUAL "ON")
        )
 endif()
 
+# LLVM
+# TODO(yueyang): Once the JIT function is fully implemented and stable, remove unnecessary libraries.
+set (LLVM_LIBRARIES
```

Comment:
Add build parameter to control this?

Reply:
Some libraries may not be needed initially. Maybe I can start by adding one library and add others as needed during the feature implementation. This approach minimizes the impact on the existing program.